src/replica/replication_app_base.h

```diff
@@ -85,6 +85,26 @@ class replica_app_info
     error_code store(const std::string &fname);
 };
 
+// This class stores and loads EEK, IV, and KV from KMS as a JSON file.
+// To get the decrypted key, should POST EEK, IV, and KV to KMS.
+class replica_kms_info
```

Comment:
How about defining it as a struct, then the "public" words can be omitted.

Reply:
```suggestion
class kms_info
```
Because this is not a "replica" level info, it's a server wide info.